jax/_src/scipy/linalg.py

```diff
@@ -271,7 +306,11 @@ def svd(a: ArrayLike, full_matrices: bool = True, compute_uv: bool = True,
       only the singular values ``s``.
     overwrite_a: unused by JAX
     check_finite: unused by JAX
-    lapack_driver: unused by JAX
+    lapack_driver: On CPU target the parameter selects the SVD algorithm. The
+      default option 'gesdd' selects the generally more efficient
+      divide-and-conquer LAPACK method, the other valid parameter 'gesvd'
+      selects the QR based algorithm. On other targets, the parameter is
+      ignored.
```

Comment:
I'm not convinced that we want to include this parameter here. I see that this is the scipy API, but I'm not sure that we want to have platform-specific parameters like this here. I think that my preference would be to have users who need algorithm control use the `lax` API instead of this one. What do you think?

Reply:
I just encountered some `NaN`s resulting from non-converging SVDs and only knew to check the QR-based driver because I had talked to @JanLuca about this issue in the past.

I don't have an opinion on whether the driver should be made available here. But if it ends up only in the `lax` API, in my opinion it would be helpful to add a hint here, to point users towards `lax.linalg.svd` and let them know that this might resolve their `NaN`-problems.